src/routes/DigitalAssets/routes/Grid/DigitalAssetsGridView.js

```diff
@@ -18,15 +22,14 @@ const DIGITAL_ASSETS_TABS = {
 type Props = {|
   openView: () => void,
   closeView: () => void,
-  setSearchQuery: (string) => void,
+  addAssetClick: () => void,
```

Comment:
Future suggestion: there is could rule in eslint react plugin that suggests using `handle` (or any other) prefix to differ between DOM event handlers and other methods. Maybe we should consider using it?